app/controllers/clusters_controller.rb

```diff
@@ -82,10 +82,16 @@ class ClustersController < AuthenticatedUsersController
   post '/clusters/:cluster_id/unmanage' do
     Tendrl.load_node_definitions
     flow = Tendrl::Flow.new('namespace.tendrl', 'UnmanageCluster')
+    body = JSON.parse(request.body.read)
+    body['Cluster.delete_telemetry_data'] = if ['yes', 'no'].include?(body['Cluster.delete_telemetry_data'])
```

Comment:
The default should be "yes" if no value is supplied

Reply:
Deletion if any data is disruptive action so I thought of keeping default as archival.
@julienlim @nthomas-redhat thoughts?

Reply:
@shtripat We suggested in the design for the default to be to delete the data.  That way, if the user wants to retain/archive the data, it's a conscious decision to do so.

Reply:
@julienlim thanks for confirmation. I will change the logic accordingly.
@r0h4n thanks for pointing this out.